include/VCR_Globals.h

```diff
@@ -17,8 +17,19 @@ extern VCRData_s vcr_data; // NOLINT
 
 /* ADC setup */
 constexpr unsigned int channels_within_mcp_adc = 8;
+struct ADCInstance
+{
+    MCP_ADC <channels_within_mcp_adc> & adc0;
+    MCP_ADC <channels_within_mcp_adc> & adc1;
+    
+    ADCInstance(MCP_ADC <channels_within_mcp_adc> & adc0_, MCP_ADC <channels_within_mcp_adc> & adc1_) : adc0(adc0_), adc1(adc1_) {};
+    // ADCInstance() : adc0(adc0_), adc1(adc1_) {}; // Default constructor
+};
+using ADCSingletonInstance = etl::singleton<ADCInstance>; // Singleton for ADCs. Used to pass ADCs to other systems that need them, such as the TelemetrySystem.
```

Comment:
Then, in accordance with all other etl::singleton uses, name this "<classname>Instance".